src/test/java/org/wso2/carbon/connector/unit/test/ldap/LdapServerSetup.java

```diff
@@ -0,0 +1,150 @@
+package org.wso2.carbon.connector.unit.test.ldap;
+
```

Comment:
License header is missing, check this throughout the PR

Reply:
Done